src/internal/diagnostics/diagnostics.go

```diff
@@ -0,0 +1,160 @@
+package diagnostics
+
+import (
+	"context"
+	"os"
+	"runtime/trace"
+	"strings"
+
+	"github.com/aws/aws-xray-sdk-go/xray"
+	"github.com/aws/aws-xray-sdk-go/xraylog"
+	"github.com/pkg/errors"
+
+	"github.com/alcionai/corso/src/pkg/logger"
+)
+
+var localRun bool
+
+/*
+Currently using AWS x-ray for observability:
+https://docs.aws.amazon.com/xray/latest/devguide/xray-concepts.html#xray-concepts-annotations
+
+runtime/trace is also collected for load_test metrics gathering.
+*/
+
+// Initialize trace and span collection and emission.
+// Should be called as an initialization
+func InitCollector() error {
+	// TODO: ignore configuration if observability tooling (such as the x-ray daemon)
```

Comment:
Just to make sure I understand how things get initialized - is this TODO needed here or is this controlled by whoever calls `InitCollector`?

Reply:
Ah, I *think* I had that wrong.  IIRC, sdk consumers should be initializing and running their own otel collector, and shouldn't need to call this initializer.  The comment here imagined this running outside of the load tests (maybe by building it into the deployment, or some such) much like our metrics.  Doesn't seem like a point of concern at this time.